java/java.source.base/src/org/netbeans/api/java/source/SourceUtils.java

```diff
@@ -1477,10 +1478,11 @@ public static void forceSource(CompilationController cc, FileObject file) {
      *
      * @param info the ClasspathInfo used to resolve
      * @param relativePath input source file path relative to the corresponding source root
+     * @param nestedClass nested class which name is searched
      * @return class name for the corresponding input source file
      * @since 2.73
      */
-    public static String classNameFor(ClasspathInfo info, String relativePath) {
+    public static String classNameFor(ClasspathInfo info, String relativePath, NestedClass nestedClass) {
```

Comment:
Sorry I missed this one. Paperwork for this API addition is missing - `apichanges.xml` entry + update version for  dependency on `java.source.base' in maven + gradle modules which depend on this new API